tests/dev-requirements.txt

```diff
@@ -15,3 +15,8 @@ pyzmq==15.1.0
 python-dateutil
 wagon
 elasticsearch==1.6.0
+psycopg2==2.6.2
```

Comment:
Are you using all of these packages?
I couldn't find a place where webargs is used...